compiler/stmt.py

```diff
@@ -41,6 +63,7 @@ def __init__(self, block_):
     self.block = block_
     self.writer = util.Writer()
     self.expr_visitor = expr_visitor.ExprVisitor(self.block, self.writer)
+    self.parser_flags = 0
```

Comment:
I'm guessing that the ast module doesn't check that `from __future__` imports satisfy these requirements so I think we should add a check and raise ParseError when `from __future__` appears later in the file.

Not sure the best way to do this. Maybe override NodeVisitor.visit()?